terraform/environment/sbox/21-bastion.tf

```diff
@@ -38,4 +38,5 @@ module "bastion-secops-sbox" {
   # Tenable Nessus
   nessus_server     = "nessus-scanners-sbox000006.platform.hmcts.net"
   nessus_key_secret = "nessus-agent-key-sbox"
+  common_tags       = merge(var.common_tags, local.auto_shutdown_common_tags)
```

Comment:
var.common_tags here is an empty map, this should reference a call to the common tags module